cli/src/commands/graph/monograph/commands/create.ts

```diff
@@ -24,6 +29,10 @@ export default (opts: BaseCommandOptions) => {
     '--subscription-protocol <protocol>',
     'The protocol to use when subscribing to the graph. The supported protocols are ws, sse, and sse_post.',
   );
+  command.option(
+    '--websocket-subprotocol <protocol>',
+    'The subprotocolto use when subscribing to the subgraph. The supported protocols are auto, graphql-ws, and graphql-transport-ws.',
```

Comment:
"The subprotocolto" typo
Please explain "auto" or link to more info.
State what the default protocol is.